src/components/TableComponent/index.jsx

```diff
@@ -1,166 +1,120 @@
-import React from 'react';
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
This ESLint rule generally should not be ignored.

Reply:
its an old PR but as far as i remember, by adding those dependencies, the actual functionality was breaking. that's why i have added this ignore statement